main.go

```diff
@@ -1,9 +1,91 @@
 package main
 
 import (
-	"github.com/testifysec/archivista-data-provider/cmd"
+	"crypto/tls"
+	"crypto/x509"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/testifysec/archivista-data-provider/pkg/handler"
+	"github.com/testifysec/archivista-data-provider/pkg/manager"
+	"github.com/testifysec/go-witness/archivista"
+
+	"k8s.io/klog/v2"
+)
+
+const (
+	// TODO: Fix timeout handling.
+	timeout     = 300 * time.Second
+	defaultPort = 8090
+
+	certName = "tls.crt"
+	keyName  = "tls.key"
 )
 
+var (
+	certDir      string
+	clientCAFile string
+	port         int
+)
+
+func init() {
+	klog.InitFlags(nil)
+	flag.StringVar(&certDir, "cert-dir", "", "path to directory containing TLS certificates")
+	flag.StringVar(&clientCAFile, "client-ca-file", "", "path to client CA certificate")
+	flag.IntVar(&port, "port", defaultPort, "Port for the server to listen on")
+	flag.Parse()
+}
+
 func main() {
-	cmd.Execute()
+	ac := archivista.New("https://archivista.testifysec.io")
```

Comment:
do we want this to be configurable someday so that people can specify another URL, or many URLs?